Yoast/YoastSEO.js
# Refactor keyphrase density to include Recalibration toggle

## Summary

This PR can be summarized in the following changelog entry:

* Introduces a recalibrated version of the keyhrase density assessment, which changes scoring schema to account for the length of the keyphrase and changes feedback strings.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Use the new example to check if two versions (recalibration and regular) return meaningful results.
* For regular analysis: check if the results are no different from `9.1`.
* For recalibration analysis:
  * Check if the new feedback strings are returned. They should not give away the actual keyphrase density, but talk about good number of matches for the text length.
  * Check if the results change depending on the length of the keyphrase, i.e. with the same actual keyphrase density the results would be different for a keyphrase that consists of 2 words and the one the consists of 7 words.
    * If the text consists of 1000 words and you used your 2-words keyphrase 20 times you should get a green score.
    * If the text consists of 1000 words and you used your 5-words keyphrase 20 times you should get a red score.

Fixes #1924
